renovate.json

```diff
@@ -24,6 +24,12 @@
 		"node",
 		"npm"
 	],
+	"vulnerabilityAlerts": {
+		"dependencyDashboardApproval": false,
```

Comment:
Isn't it `false` [by default](https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval)?

Reply:
Good catch